### PR TITLE
fix(Select): Text vertical alignment

### LINKED ISSUE #1639: Select: Text vertical alignment changes when a value is selected
Noticed in engage.

Select label is vertically aligned correctly with icons UNTIL a selection is made, then the text is too low:

### PR DESCRIPTION
**Pull request by atlwendy**
closes #1639 

image

### CODE REVIEW

**Comment by benjamincharity** [comment]
@atlwendy for UI changes like this, can you upload a screenshot/gif of the fix?

File: terminus-ui/select/src/select.component.scss [hunk 1]
```diff
@@ -25,6 +25,12 @@ $ts-select-placeholder-arrow-space: 2 * ($ts-select-arrow-size + $ts-select-arro
   @include reset;
   display: block;
   outline: none;
+  .ts-form-field {
+    .ts-form-field__infix {
+      $select-form-field-padding: .6em;
```
Comment (benjamincharity) [inline terminus-ui/select/src/select.component.scss]: We should probably either a) add `-top` to to this variable name, or b) move all the padding numbers up into the definition.

**Review by benjamincharity — CHANGES_REQUESTED** [review]
Apparently this comment input is required now... lame

File: terminus-ui/select/src/select.component.scss (hunk 1 above)
Comment (atlwendy) [reply terminus-ui/select/src/select.component.scss]: I was actually wondering whether you'd call this out lol

Also, your overall comment made me think some about `input` component stuff is required when first glanced at it!

**Review by benjamincharity — APPROVED** [review]

**Pull request merged**